### ISSUE TITLE
tooltip no longer working (atom-beta 1.9.0, osx, psc 0.9.2)

### ISSUE THREAD

**Issue by rvion** [nwolverson/atom-ide-purescript#95]
I can't make to tooltip to work.
I think I saw it working at some point on almost empty empty projects.

nothing is visible in the logs neither, as if hovering didn't trigger anything

_btw, awesome project anyway, thanks a lot for this atom package. it's working great overall!!_

**Comment by kritzcreek** [comment]
@rvion after switching the compiler you'll most likely have to delete the output folder to clear any out-of-date build artifacts.

**Comment by nwolverson** [comment]
- Check autocomplete (probably this doesn't work also)
- Check debug console on startup - is the psc-ide-server location/version looking correct
- remove output & rebuild, then switch file or restart psc-ide-server (There's a menu option)

**Comment by rvion** [comment]
this is a fresh install :)

@nwolverson 

> Check autocomplete (probably this doesn't work also)

autocomplete is working, only hovering doesn't
![image](https://cloud.githubusercontent.com/assets/2150990/16741484/b8d748ae-47a3-11e6-90f6-aed4faaee8c7.png)

> Check debug console on startup - is the psc-ide-server location/version looking correct

it seems that everything is working
![image](https://cloud.githubusercontent.com/assets/2150990/16741534/f8823ea0-47a3-11e6-9568-98f87d3995c7.png)

also, 

``` shell
➜  psc-test $ psc-ide-server --version
0.9.2
➜  psc-test $ psc-ide-client --version
0.9.2
➜  psc-test $ psc --version
0.9.2
```

> remove output & rebuild, then switch file or restart psc-ide-server (There's a menu option)

I did so, but it doesn't fix my problem.

Also, all of my atom packages are up to date.
Could it be because of atom 1.9.0-beta0 ?

**Comment by rvion** [comment]
Tell me if I can help by giving more informations :)

**Comment by nwolverson** [comment]
Maybe worth trying switching file & back to the `.purs` file you're editing, to make it reregister that as active file / modules that are open etc.

I'll have to check against 1.9.0 & 0.9.2, but I feel like I may have seen this before temporarily. Will probably be implications of #86 too so if this isn't obvious it may have to wait for that. (Thanks for your feedback generally)

**Comment by rvion** [comment]
> Maybe worth trying switching file & back to the .purs file you're editing, to make it reregister that as active file / modules that are open etc.

@nwolverson:  it's not working

📝  but here are some interesting informations related to my experiments:

I created a new project 

```
cd $dev
mkdir xxxx
cd xxxx
pulp init
```

I killed the previous psc-ide
started `atom-beta .`

and.... magic: it works in the new project. 🍏 

So I thought it could also be fixed in the first project. I killed the sever, opened back the first project, but no 🔴  the initial project still doesn't have tooltip working.
I nuked most folders (output, pulp cache, etc.) but it still doesn't work in the first project, and still work for the new one.

**Comment by rvion** [comment]
@nwolverson Ahah ! I found something that could help you understand why it's happening:

in https://github.com/nwolverson/atom-ide-purescript/issues/101, I reported a problem where I have 2 modules, but only one triggers type-checking uppon save, while the other doesnt.

I just noticed that **the file that triggers re-typechecking has tooltips, while the other hasn't.** I think this is related to the same root cause. At some point, atom-ide-purescript decided that some file was "not part of the app anymore".

Also, as mentionned above and in https://github.com/nwolverson/atom-ide-purescript/issues/101, restarting the ide-server doesn't change anything/
